src/server/src/tests/deck.ts

```diff
@@ -0,0 +1,22 @@
+import { expect } from 'chai';
```

Comment:
this shouldn't show as part of this PR. try running a `git pull origin master`

Reply:
Ran it but it says that im up to date. I must have merged incorrectly somehow.

Reply:
oh wait this shouldn't be here. this should be called `deck.test.ts`